R/fiveyeargwsum-styles.R

```diff
@@ -1,12 +1,20 @@
-getFiveyearStyle <- function(data, info=NULL) {
+getFiveyearStyle <- function(data, info=NULL, invertedPlot = FALSE) {
 
     x <- data[[1]]$time
     y <- data[[1]]$value
+    
+    #semantics for min/max are swapped on inverted plots
+    minMaxLabels = c("Max. Instantaneous", "Min. Instantaneous");
+    if(invertedPlot) {
+      minMaxLabels = c("Min. Instantaneous", "Max. Instantaneous");
+    }
+    
```

Comment:
I'm not sure I consider this data munging, it's strictly a labeling change. That is unless we consider the legend title to be part of the data, is that how we are treating it else where?